.github/workflows/t3000-unit-tests.yaml

```diff
@@ -22,6 +22,7 @@ jobs:
           { name: "t3k falcon7b tests", arch: wormhole_b0, cmd: run_t3000_falcon7b_tests, timeout: 30, owner_id: UBHPP2NDP}, #Joseph Chu
           { name: "t3k falcon40b tests", arch: wormhole_b0, cmd: run_t3000_falcon40b_tests, timeout: 30, owner_id: U044T8U8DEF}, #Johanna Rock
           { name: "t3k mixtral tests", arch: wormhole_b0, cmd: run_t3000_mixtral_tests, timeout: 30, owner_id: U03PUAKE719}, #Miguel Tairum Cruz
+          { name: "t3k grok tests", arch: wormhole_b0, cmd: run_t3000_grok_tests, timeout: 30, owner_id: U03PUAKE719}, #Miguel Tairum Cruz
```

Comment:
will need a slack owner here. Are you the owner @yieldthought (your slack member ID is U03HY7MK4BT)
Owners are pinged using their own slack ID and are pinged in a slack chat if their test fails on main branch.
https://forums.slackcommunity.com/s/question/0D5Hq00009mZfnuKAC/hello-i-am-a-slack-newbie-how-do-i-find-my-slack-id?language=en_US

Reply:
Added my slack ID, thanks.